src/config/mobileG5.js

```diff
@@ -47,10 +47,12 @@ const fennec64onP2 = {
     [1071.28, 664.5, 49.42, 716.5, 64.35, 2581, 72.18, "Tp6 mobile: Aframe.io", "https://aframe.io/examples/showcase/animation"],
   ]
 };
+// Initial platform was geckoview on p2-aarch64
 fennec64onP2.header.push("platform");
 fennec64onP2.data.forEach(d=>d.push('geckoview-p2-aarch64'));
 const flatP2 = fennec64onP2.data.map(row => Data.zip(fennec64onP2.header, row));
-
+// ADD fenic ON P2
```

Comment:
Nitpicky typo in comment: should read 'fenix' (as it reads it's easily confused with 'fennec')